app/components/Counter/tests/index.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render } from 'react-testing-library';
+import { Provider } from 'react-redux';
+import { IntlProvider } from 'react-intl';
+import { ConnectedRouter } from 'connected-react-router/immutable';
+import { createMemoryHistory } from 'history';
+
+import Header from '../index';
+import configureStore from '../../../configureStore';
+
+describe('<Header />', () => {
```

Comment:
Header ;) ?

Reply:
Right...! #facepalm This component was created based on Header and I forgot about it with test. I fixed it and removed all test files for now.